definitions/ext-snowflake_account/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: SNOWFLAKEACC
+
+synthesis:
+  identifier: snowflakeAccount
+  name: snowflakeAccount
+
+  conditions:
+    # The attribute’s value must match the provided value
+    - attribute: event_type
```

Comment:
```suggestion
    - attribute: eventType
```

Reply:
This one is still pending to fix @danifitz